Cargo.toml

```
@@ -1,9 +1,18 @@
 [package]
 name = "eslauncher2"
 version = "0.4.5"
-authors = ["M*C*O <mcofficer@gmx.de>"]
+authors = ["EndlessSkyCommunity"]
```

I'm not sure we should imply that the ESC org is the primary maintainer or original author of this package.

---

I was following MCO's suggestion. The author field does allow for a list authors, so we could add everyone who has contributed instead. 
@MCOfficer - what is your take on this?

---

the author fields allows for multiple values, anything against going with both? have me as primary maintainer because i wrote it, and ESC as secondary, because that's the org this repo belongs to.
```suggestion
authors = ["M*C*O <mcofficer@gmx.de>", "EndlessSkyCommunity"]
```

---

I'll go with this one, then. Makes the most sense to me